main.py

```diff
@@ -1,7 +1,48 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors_name = ["George", "Kianu", "Scarlett",
+                   "Will", "Jaden", "Scarlett"]
+    actors_surname = ["Klooney", "Reaves", "Keegan",
+                      "Smith", "Smith", "Johansson"]
```

Comment:
I recommend you create one variable "actors" with a list of tuples [("George", "Klooney"), ...]. And after that in for loop use this variable. Also its recommendation from the checklist

Reply:
![ThumbsUpSimonCowellGIF](https://github.com/user-attachments/assets/9c11e70e-1675-4fae-ae80-83840782c2fb)